lib/kafo/kafo_configure.rb

```diff
@@ -166,6 +166,12 @@ def logger
     def run(*args)
       started_at = Time.now
       logger.debug("Running installer with args #{args.inspect}")
+      if config.app[:verbose]
+        log_output_message = <<~MESSAGE.strip
+          Running installer with log based terminal output at level #{config.app[:verbose_log_level].upcase}. Use -l to change output verbosity to one of ERROR, WARN, NOTICE, INFO, or DEBUG.
```

Comment:
Does the option handle uppercase log levels? I feel like it should but I cannot say I have ever tried it.

Reply:
Yes, it works fine in my testing to run `foreman-installer -l INFO` for example

Reply:
Looks like it's normalized here:
https://github.com/TwP/logging/blob/ae9872d093833b2a5a34cbe1faa4e895a81f6845/lib/logging.rb#L513